### DISCUSSION TITLE
Button success state and errors

### DISCUSSION THREAD

**Discussion by codercody** [mantinedev/mantine#6659] [Feature requests]
Hi, I'm a big fan of the loading state for Button. It would also be great if there could be a success state that is similar to the loading state: disabled, a checkmark in the center, and that little slide-in transition the loading state has. Thanks!

**Comment by rtivital** [comment]
Button already has disabled state. If you want to customize disabled styles, use Styles API – https://mantine.dev/core/button/#disabled-state

**Comment by codercody** [comment]
Right, I know it's possible to do manually, but it's a pretty common use case for me and I'd imagine others too: click a button, set it to loading, wait for some server response, set it to success with a checkmark.
